lib/augmentConfig.js

```diff
@@ -67,7 +67,8 @@ function augmentConfigExtended(
 	if (!cosmiconfigResult) return Promise.resolve(null);
 
 	const configDir = path.dirname(cosmiconfigResult.filepath || '');
-	const cleanedConfig = _.omit(cosmiconfigResult.config, 'ignoreFiles');
+	// eslint-disable-next-line no-unused-vars
```

Comment:
Is there not something we can do with ESLint here instead of this?

Reply:
How about using the [`ignoreRestSiblings`](https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings)?

```js
/*eslint no-unused-vars: ["error", { "ignoreRestSiblings": true }]*/
// 'type' is ignored because it has a rest property sibling.
var { type, ...coords } = data;
```

Reply:
No idea, I don't want to spend my time fighting with linting tools :P

That being said, if everyone else agrees with your suggestion, feel free to push it to this branch :)

Reply:
I have no strong opinion about this. 😃 
I think that disabling via the magic comment is a low cost solution.

Reply:
Feel free to add this in this branch or in the ESLint config here or upstream :)

Reply:
Thanks! I pushed 1046a9896284e9535065bc804f99211444c35721.

Reply:
Should the suppressions be at the top of the files, though?

Reply:
Maybe yes. I read the ESLint document, but "Using Configuration Comments" does not seem to support code blocks to me.

https://eslint.org/docs/user-guide/configuring#using-configuration-comments

Reply:
No worries, then. Maybe this should be in our ESLint config, though. Let's wait for others to chime in.